SIT.Manager.Avalonia/ViewModels/InstallPageViewModel.cs

```diff
@@ -42,9 +34,45 @@ public partial class InstallPageViewModel : ObservableRecipient,
     [ObservableProperty]
     private ReadOnlyCollection<InstallStep> _installationSteps;
 
-    public InstallPageViewModel()
+    public InstallPageViewModel(ILocalizationService localizationService, IManagerConfigService managerConfigService)
     {
+        _managerConfigService = managerConfigService;
+        _localizationService = localizationService;
+        _managerConfigService.ConfigChanged += (o, e) =>
+        {
+            _sitInstallSteps =
+            [
+                new(typeof(SelectView), _localizationService.TranslateSource("InstallPageViewModelSelectText")),
+                new(typeof(ConfigureSitView), _localizationService.TranslateSource("InstallPageViewModelConfigureText")),
+                new(typeof(PatchView), _localizationService.TranslateSource("InstallPageViewModelPatchText")),
+                new(typeof(InstallView), _localizationService.TranslateSource("InstallPageViewModelInstallText")),
+                new(typeof(CompleteView), _localizationService.TranslateSource("InstallPageViewModelCompleteText"))
+            ];
+            _serverInstallSteps =
+            [
+                new(typeof(SelectView), _localizationService.TranslateSource("InstallPageViewModelSelectText")),
+                new(typeof(ConfigureServerView), _localizationService.TranslateSource("InstallPageViewModelConfigureText")),
+                new(typeof(InstallView), _localizationService.TranslateSource("InstallPageViewModelInstallText")),
+                new(typeof(CompleteView), _localizationService.TranslateSource("InstallPageViewModelCompleteText"))
+            ];
+            InstallationSteps = _sitInstallSteps.AsReadOnly();
+        };
         WeakReferenceMessenger.Default.RegisterAll(this);
+        _sitInstallSteps =
+        [
+            new(typeof(SelectView), _localizationService.TranslateSource("InstallPageViewModelSelectText")),
+            new(typeof(ConfigureSitView), _localizationService.TranslateSource("InstallPageViewModelConfigureText")),
+            new(typeof(PatchView), _localizationService.TranslateSource("InstallPageViewModelPatchText")),
+            new(typeof(InstallView), _localizationService.TranslateSource("InstallPageViewModelInstallText")),
+            new(typeof(CompleteView), _localizationService.TranslateSource("InstallPageViewModelCompleteText"))
+        ];
+        _serverInstallSteps = 
+        [
+            new(typeof(SelectView), _localizationService.TranslateSource("InstallPageViewModelSelectText")),
+            new(typeof(ConfigureServerView), _localizationService.TranslateSource("InstallPageViewModelConfigureText")),
+            new(typeof(InstallView), _localizationService.TranslateSource("InstallPageViewModelInstallText")),
+            new(typeof(CompleteView), _localizationService.TranslateSource("InstallPageViewModelCompleteText"))
+        ];
```

Comment:
Not a fan of these updating on *any* config change

Reply:
> Not a fan of these updating on _any_ config change

There is no other way to translate the steps.

Reply:
Given how often this is an issue you could probably create a new event in the config service for when the localization changes

Reply:
> Given how often this is an issue you could probably create a new event in the config service for when the localization changes

Good point